nix/animView.nix

```diff
@@ -0,0 +1,54 @@
+{ baseName ? "corsix-th", stdenv, lib, cmake, wxGTK30 }:
```

Comment:
Please don't do it this way. cmake the root, then make AnimView to build your target.